jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/ExplorerNodes.kt

```diff
@@ -19,7 +19,7 @@ import software.aws.toolkits.jetbrains.core.explorer.AwsTruncatedResultNode
 import software.aws.toolkits.resources.message
 
 class LambdaServiceNode(project: Project) :
-    AwsExplorerServiceRootNode(project, message("lambda.service_name"), AwsIcons.Logos.LAMBDA) {
+    AwsExplorerServiceRootNode(project, message("lambda.service_name"), null) {
```

Comment:
If we are going to keep this interface we might as well give `awsIcon` a default value of `null` rather than having to pass it explicitly.